opentech/apply/funds/templates/funds/base_submissions_table.html

```diff
@@ -17,6 +17,7 @@
 
 {% block extra_js %}
     {{ filter.form.media.js }}
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/iamdustan-smoothscroll/0.4.0/smoothscroll.min.js"></script>
```

Comment:
Is this required still? and do we want it as a lib in package if including?

Reply:
nope, will remove

Reply:
still required to polyfill `element.scrollTo` in ms browsers, but will move it to be added as a package